package.json

```diff
@@ -200,18 +200,28 @@
   "scripts": {
     "start": "stripes serve",
     "build": "stripes build --output ./output",
-    "test": "stripes test nightmare --run demo --show",
+    "test": "stripes test karma",
     "lint": "eslint ."
   },
   "devDependencies": {
+    "@bigtest/interactor": "^0.9.0",
+    "@bigtest/mirage": "^0.0.1",
+    "@bigtest/mocha": "^0.5.1",
+    "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^3.2.1",
     "@folio/stripes": "^1.0.0",
+    "@folio/stripes-cli": "^1.6.0",
+    "@folio/stripes-core": "^2.15.4000415",
```

Comment:
Please use version `^12.15.4` of stripes-core.

This very large patch segment (`4000415`) is a sign that your dependency was added against the FOLIO CI registry, `npm-folioci`.  Unfortunately these inflated patch version numbers are specific to `npm-folioci` and not semver compatible with the `npm-folio` registry.  All `package.json` folio-scoped dependencies should reference versions available in `npm-folio`.